repo-client/src/main/java/io/lionweb/repoclient/LionWebRepoClient.java

```diff
@@ -112,7 +110,7 @@ public LionWebRepoClient(
       @NotNull LionWebVersion lionWebVersion,
       @NotNull String hostname,
       int port,
-      @Nullable String authorizationToken,
+      @Null String authorizationToken,
```

Comment:
I think `@Null` has different meaning. Quoting from [docs](https://docs.oracle.com/javaee/7/api/javax/validation/constraints/Null.html):

> The annotated element must be null.

Reply:
You are right, I am reverting back to the jetbrains' annotations